lms/djangoapps/hera/admin.py

```diff
@@ -13,7 +13,29 @@ class OnboardingAdmin(admin.ModelAdmin):
     """
     Reflection of the hera model on the admin panel.
     """
-    pass
+
+    def get_queryset(self, request):
+        qs = super(OnboardingAdmin, self).get_queryset(request)
```

Comment:
Also need to add docstring for this case.

Reply:
Done.